posts/automated-api-testing-with-postman.md

```diff
@@ -0,0 +1,169 @@
+---
+authors:
+- Matthew Reed
+- Amine Ouali Alami
+tags:
+- API
+- TDD
+- Testing
+- Postman
+
+date: 2020-09-26T20:16:10.000Z
+title: "Start Automating API Tests with Postman"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/10/postman_logo.png
+---
+
+## What is Wrong with Cucumber
```

Comment:
```suggestion
## What's Wrong with Cucumber?
```

Reply:
Had that before but expanded because README states to `Avoid abbreviations (e.g. "I've" -> "I have")`